ledger/apply/payment_test.go

```diff
@@ -111,8 +111,10 @@ func TestPaymentApply(t *testing.T) {
 func TestCheckSpender(t *testing.T) {
 	partitiontest.PartitionTest(t)
 
-	mockBalV0 := makeMockBalances(protocol.ConsensusCurrentVersion)
 	mockBalV7 := makeMockBalances(protocol.ConsensusV7)
+	mockBalV39 := makeMockBalances(protocol.ConsensusV39)
+	mockBalCurrent := makeMockBalances(protocol.ConsensusCurrentVersion)
```

Comment:
This will stop working when we cut the release. Is that the intent? Maybe a "TODO: replace this with the first consensus version with incentives" would be welcome reminder in that case

Reply:
"intent" is maybe too strong of a word. I couldn't think of a better way to write it to encode what we want.  Except by deleting `mockBalCurrent` completely.  I would do that, but it seemed like that might get me some spicy reactions.

Reply:
Reviewers, please vote: add a comment, or stop testing "current".

I lean toward removing the "current" check and only do a "future" check. (v39 will stay)

Reply:
I've committed a change that tests v39 and future, but not current.

Reply:
I think future is how we had handled this situation elsewhere cc @cce

Reply:
okay I'm good with the last commit.

Reply:
Latest change looks good to me